githubbot/githubbot/handler.go

```diff
@@ -35,7 +35,7 @@ func NewHandler(stats *base.StatsRegistry, kbc *kbchat.API, debugConfig *base.Ch
 	httpPrefix, appName string) *Handler {
 	return &Handler{
 		DebugOutput: base.NewDebugOutput("Handler", debugConfig),
-		stats:       stats,
+		stats:       stats.SetPrefix("Handler"),
```

Comment:
if we're going to do this can you add it throughout the handlers/http servers?

Reply:
@joshblum done 👍